src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "bob";
+    private static final String VALID_PASSWORD = "bob123";
+    private static final int VALID_AGE = 42;
+    private static final String INVALID_PASSWORD = "admin";
+    private static final int INVALID_AGE = 17;
+
+    private RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+        user = new User();
+        user.setLogin(VALID_LOGIN);
+        user.setPassword(VALID_PASSWORD);
+        user.setAge(VALID_AGE);
+    }
+
+    @Test
+    public void register_allCorrect_ok() {
+        assertEquals(registrationService.register(user), user);
+    }
+
+    @Nested
+    public class ExceptionExpectingTests {
+        @Test
+        public void register_nullUser_notOk() {
+            user = null;
+        }
+
+        @Test
+        public void register_userAlreadyExists_notOk() {
+            System.out.println(Storage.people);
+            User user2 = new User();
+            user2.setLogin(VALID_LOGIN);
+            user2.setPassword(VALID_PASSWORD);
+            user2.setAge(VALID_AGE);
+            registrationService.register(user2);
+            assertThrows(RuntimeException.class, () -> {
+                registrationService.register(user);
+            });

```

Comment:
```suggestion
            assertThrows(RuntimeException.class, () -> 
                registrationService.register(user)
            );
```
давай або перевіряти виклик помилки після кожного методу @AfterEach або прописувати помилку в кожному тестовому методі (я за другий варіант, оскільки зазвичай для кожного тестового методу прописують повідомлення чому тест не проходить)